backend/routes/reservation.js

```diff
@@ -89,7 +87,8 @@ router.get('/', async (req, res) => {
  *
  * /reservation/restaurant:
  *   get:
- *     description: Fetch all reservations for a restaurant
+ *     tags: [Reservation]
+ *     description: Fetch a all reservation for a restaurant
```

Comment:
I think you overrode changes for the description, and for the POST and PUT endpoints too. Can you please fix this?